flink-table/flink-sql-parser/src/main/codegen/data/Parser.tdd

```diff
@@ -578,7 +579,6 @@
     "OVERWRITING"
     "PARTITIONED"
     "PARTITIONS"
-    "TRY_CAST"
```

Comment:
I am curious why we are removing this. I see [Calcite Jira ](https://issues.apache.org/jira/browse/CALCITE-4771?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=17414749)  refers to try cast and targets Calcite 1.35.